verilog/analysis/verilog_linter.cc

```diff
@@ -228,19 +256,36 @@ ViolationFixer::AnswerChoice ViolationFixer::InteractiveAnswerChooser(
       "P - show fixes applied in this file so far\n"
       "? - print this help and prompt again\n";
 
+  // If we have alternatives, show this in the short-menu.
+  std::string alternative_list;
+  if (size_t fix_count = violation.autofixes.size(); fix_count > 1) {
+    for (size_t i = 0; i < fix_count; ++i)
+      alternative_list += std::to_string(i + 1) + ",";
+  }
+
   for (;;) {
     const char c = verible::ReadCharFromUser(
         std::cin, std::cerr, verible::IsInteractiveTerminalSession(),
-        "Autofix is available. Apply? [y,n,a,d,A,D,p,P,?] ");
+        "Autofix is available. Apply? [" + alternative_list +
+            "y,n,a,d,A,D,p,P,?] ");
 
     switch (c) {
+      case '1':
```

Comment:
I think it might be cleaner to handle numeric inputs in a separate construct, outside of (after) this case statement.  It feels awkward to have to enumerate and limit choices to 3 even if that is sufficient in practice.

Reply:
I wanted to avoid expanding the notion of [AnswerChoice](https://github.com/chipsalliance/verible/pull/939/files#diff-1a62db1c3e48ce6fae3dbe0110a88c70bdbb2cd3e5b71fdf4e4769a26b6440d5R174-R177) from an enum to something else though I could be convinced otherwise (but it would be overkill at this point).

I could imagine something like the following to return from this chooser
```
  struct Answer {
     AnswerChoice choice;
     int alternative = 0;
 }
```

It _could_ be useful for the 'apply all' situation to also convey the potential alternatives.

Then, when we have such a struct, it would also warrant making this switch/case more specific if it sees some digit.

Reply:
Alright, changed the Answer to be a more complex struct like suggested above to convey the alternative.

However, this switch/case didn't change much, as we still handl just single-character interactions (though added TODO to potentially reconsider that for an 'apply all always the second alternative' kind-of situations)